spec/apps/register-with-object/register-with-object.spec.js

```diff
@@ -0,0 +1,65 @@
+let bootstrapped, mounted;
+
+const app = {
+  bootstrap() {
+    bootstrapped = true;
+    return Promise.resolve();
+  },
+  mount() {
+    mounted = true;
+    return Promise.resolve();
+  },
+  unmount() {
+    mounted = false;
+    return Promise.resolve();
+  },
+};
+
+export default function() {
+	describe(`register-with-object`, () => {
+		beforeAll(() => {
```

Comment:
Mixture of tabs and spaces.

We should convert everything to spaces across the whole project, but if that isn't done in this pr we should at least not mix tabs and spaces in new code.

Reply:
Fixed. It's all spaces now.